build.gradle

```diff
@@ -70,18 +70,14 @@ dependencies {
     compile group: 'org.scala-lang', name: 'scala-reflect', version: scalaVersion
     compile group: 'org.scala-lang', name: 'scala-compiler', version: scalaVersion
     compile group: 'org.scala-lang.modules', name: 'scala-java8-compat_2.12', version: scalaJava8CompatVersion
-    compile group: 'com.jsuereth', name: 'scala-arm_2.12', version: '2.0'
-    compile group: 'org.scala-lang.modules', name: 'scala-java8-compat_2.12', version: '0.9.1'
-
+    compile group: 'com.jsuereth', name: 'scala-arm_2.12', version: scalaArmVersion
```

Comment:
Is this a must-have dependency? I think you newly add this for some reason, just wondering where we actually need this.

Reply:
Scala Arm is required and has been used for almost 2 years since commit https://github.com/pravega/spark-connectors/commit/40e26ba0554268bdc4d0d40aa406e37176224356.

Reply:
i didnt add it, but just replace "2.0" with scalaArmVersion and add scalaArmVersion in the property file as you requested.